src/components/@shared/FormFields/URLInput/index.module.css

```diff
@@ -9,4 +9,6 @@
 
 .error {
   composes: error from '@shared/FormInput/index.module.css';
+  position: absolute;
```

Comment:
please no more `absolute` positioning hacks in this place, it is impossible to predict the exact size of the help text block as there are too many viewport + text size + text length combinations. I only used it in the formerly linked place because it was pretty much guaranteed that "Default provider" winkle stay on one text line in almost all viewports. If you can't guarantee this, never start using `position: absolute` to take elements out of box model flow.

As written before, we need a new layout here with more elements but don't know yet what exactly. Goal is to have another `position: relative` wrapper the error is attached to, but not the help text.

Case in point, with this change, this is what I get on mobile, again, impossible to deal with this in absolute positioning and percentage values:

<img width="368" alt="Screen Shot 2022-04-20 at 21 14 41" src="https://user-images.githubusercontent.com/90316/164315315-e39700cf-47d4-4d5a-81f5-e2c6d27e994c.png">